spyder/app/mainwindow.py

```diff
@@ -2405,8 +2405,19 @@ def render_issue(self, description='', traceback=''):
         if versions['revision']:
             revision = versions['revision']
 
-        # Make a description header in case no description
-        # is supplied
+        # Store and format the reminder message for the troubleshooting guide
+        reminder_message = (
+            "**PLEASE READ:** Before submitting here, please carefully "
+            "consult our **Troubleshooting Guide**: <{0}> and search the "
+            "issue page for your error/problem, as most "
+            "posted bugs are duplicates or easy fixes.\n\n"
+            "If you don't find anything, enter a step-by-step description "
+            "(in English) of what led up to the problem; issue reports "
+            "without a clear way to reproduce them will be closed. Thanks.\n\n"
+            "**Note:** Please delete this message before submitting."

```

Comment:
As I mentioned above, this should be in a markdown comment.

Reply:
See above.